sklearn/base.py

```diff
@@ -516,13 +546,30 @@ def score(self, X, y=None):
         pass
 
 
-###############################################################################
 class MetaEstimatorMixin(object):
+    _required_parameters = ["estimator"]
+
     """Mixin class for all meta estimators in scikit-learn."""
-    # this is just a tag for the moment
 
 
-###############################################################################
+class MultiOutputMixin(object):
+    """Mixin to mark estimators that support multioutput."""
+    def _get_tags(self):
+        return _update_tags(self, super(MultiOutputMixin, self),
+                            multioutput=True)
+
+
+def _is_32bit():
+    """Detect if process is 32bit Python."""
+    return struct.calcsize('P') * 8 == 32
+
+
+class _UnstableOn32BitMixin(object):
```

Comment:
I keep going back and forth, but it's no big deal.

On one hand we want contrib packages to inherit from these mixins. And that includes if they are unstable on 32bit because of arpack.

However there's a difference between functionality/intentional tags (e.g. I'm a regressor) vs this kind of technical tag. In this case maybe we  **don't want** contrib packages to inherit from this, because the meaning is not so clear and it's a bit internal.

Was this your reasoning?